internal/metadata/clusters/azure-aks.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: move2kube.io/v1alpha1
```

Comment:
One more small change @akhil-ghatiki We have change the domain name in the master branch. So for new providers, we can possibly use the one from master. But for this change if we can change the domain to "move2kube.konveyor.io" that should fix it.

Reply:
Changed this and updated to new domain name